teflo_init/execute/README.rst

```diff
@@ -0,0 +1,53 @@
+Shell
```

Comment:
What does shell mean here? Should the heading be Execute_Example ?

Reply:
fixed